### DISCUSSION TITLE
sticky-bottom : Please add a concrete example to the documentation

### DISCUSSION THREAD

**Discussion by JuliaHinssen** [twbs/bootstrap#38676] [Q&A]
Hi everybody,

my colleague and I cannot make the sticky-bottom thing work, and we also can't find an example anywhere on the internet.
Does it actually work, and if so, we would be very thankful for a working example in the documentation.

Does it also work for long content that takes up more than the monitors' height? (We have one long content, which we want to stick to the bottom, and another even longer content right next to it.)

Best wishes

Julia & Thorsten

**Comment by julien-deramond** [comment]
Hey @JuliaHinssen & Thorsten,
Seems to be related to an open issue we have on our side: https://github.com/twbs/bootstrap/issues/38111.
If so, maybe https://github.com/twbs/bootstrap/issues/38111#issuecomment-1455554130 can help you while we are tackling this issue in one of the next versions of Bootstrap. We haven't had the time to discuss it yet.

**Reply by JuliaHinssen** [reply]
Hi @julien-deramond,
I tried to apply that example with flex-utilities for our needs, but didn't find a way that worked. I am new to this, though, so there might be a way.

This is the behavior we wish to accomplish:
![Bootstrap sticky-top und sticky-bottom](https://github.com/twbs/bootstrap/assets/20791293/58dd5749-2381-4d7b-bd24-f5dc68d205a0)

The green rectangle is the area that is visible on the monitor.
 
Best wishes
Julia & Thorsten
